installation/install.sh

```
@@ -3032,14 +3032,17 @@ cs_config_zstack_properties(){
         fi
     fi
 
-    if [ ! -z $CONSOLE_PROXY_ADDRESS ];then
-        zstack-ctl configure consoleProxyOverriddenIp=${CONSOLE_PROXY_ADDRESS}
+    if [ -n "${CONSOLE_PROXY_ADDRESS}" ];then
+        if [[ "${CONSOLE_PROXY_ADDRESS}" == *":"* ]]; then
+            zstack-ctl configure consoleProxyOverriddenIpv6=${CONSOLE_PROXY_ADDRESS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)